src/utils/getDicomFilesFromDirectory.js

```diff
@@ -0,0 +1,9 @@
+import fs from "fs";
+import path from "path";
+
+export function getDicomFilesFromDirectory(directory) {
```

Comment:
You should use one of the directory tree scanners to get files in that directory and recursively, and unless the glob pattern is *.dcm, it should match everything - there might be an exclusion pattern, to exclude jpg, txt etc.